docs/sources/panels/visualizations/dashboard-list-panel.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "Dashboard list panel"
+keywords = ["grafana", "dashboard list", "documentation", "panel", "dashlist"]
+type = "docs"
+aliases = ["/docs/grafana/latest/reference/dashlist/", "/docs/grafana/latest/features/panels/dashlist/"]
+[menu.docs]
+name = "Dashboard list"
+parent = "visualizations"
+weight = 800
+draft = "true"
++++
+
+
+# Dashboard list panel
+
+The dashboard list panel allows you to display dynamic links to other dashboards. The list can be configured to use starred dashboards, recently viewed dashboards, a search query, and dashboard tags.
+
+{{< docs-imagebox img="/img/docs/v45/dashboard-list-panels.png" max-width="850px">}}
+
+On each dashboard load, this panel queries the dashboard list, always providing the most up-to-date results.
+
+## Options
+
+Use these options to refine your visualization.
+
+### Options
+
+- **Starred -** Display starred dashboards in alphabetical order.
+- **Recently viewed -** Display recently viewed dashboards in alphabetical order.
+- **Search -** Display dashboards by search query or tags. Requires you to enter at least one value in **Query** or **Tags**.
+- **Show headings -** The chosen list selection (Starred, Recently viewed, Search) is shown as a heading.
+- **Max items -** Sets the maximum number of items to list.
+
+Search
```

Comment:
I would just add that the below options are used in combination with the above search option. They won't work unless you have selected search first (sort of strange UI)